charts/images.yaml

```diff
@@ -173,7 +173,7 @@ images:
       comment: the node-exporter is also deployed to the shoot cluster
 - name: grafana
   sourceRepository: github.com/grafana/grafana
-  repository: grafana/grafana
+  repository: eu.gcr.io/gardener-project/3rd/grafana/grafana
```

Comment:
Can you extend https://github.com/gardener/gardener/blob/676ec251d1381932940be9b31f8e9e781aa62534/charts/images.yaml#L1-L5 with information that Docker Hub should be avoided because it doesn't support IPv6?